server/routes/probationPractitionerReferrals/probationPractitionerReferralsController.ts

```diff
@@ -197,6 +197,33 @@ export default class ProbationPractitionerReferralsController {
     return ControllerUtils.renderWithLayout(res, view, serviceUser)
   }
 
+  async viewSubmittedPostAssessmentFeedback(req: Request, res: Response): Promise<void> {
+    const { user } = res.locals
+    const { accessToken } = user.token
+    const referralId = req.params.id
+
+    const [referral, supplierAssessment] = await Promise.all([
+      this.interventionsService.getSentReferral(accessToken, referralId),
+      this.interventionsService.getSupplierAssessment(accessToken, referralId),
+    ])
+
+    if (!referral.assignedTo) {
+      throw new Error('Referral has not yet been assigned to a caseworker')
+    }
+
+    const { currentAppointment } = new SupplierAssessmentDecorator(supplierAssessment)
+    if (currentAppointment === null) {
+      throw new Error('Attempting to view supplier assessment feedback without a current appointment')
```

Comment:
How likely is this? If fairly likely, do we want to surface a more friendly `userMessage` here, or do you think this is enough details for users to go on? e.g. do we want to use the term "initial assessment" rather than "supplier assessment" here?

Reply:
It should not ever happen because the link would not be visible to the user until it satisfies that criteria. 
However, you're right about changing it away from supplier assessment; I'll make that change.